cmake-init/templates/common/cmake/install-rules.cmake

```diff
@@ -1,5 +1,7 @@
 {% if not exe %}if(PROJECT_IS_TOP_LEVEL)
-  set(CMAKE_INSTALL_INCLUDEDIR include/{= name =} CACHE PATH "")
+  # CMAKE_INSTALL_INCLUDEDIR is nested in one more ${PROJECT_VERSION} directory to prevent inclusion of all other headers in the install prefix directory.
+  #   See https://github.com/friendlyanon/cmake-init/issues/43 for more info
```

Comment:
Besides not keeping the 80 column limit and the period at the end, I also don't feel okay about a link back to this project.

The idea of cmake-init is that it generates projects that - in a perfect world - everyone would write to begin with, so links back feel icky to me.

Something more generic about the purpose of avoiding indirect inclusion of other projects in a shared prefix would fit better.